modal.js

```diff
@@ -29,4 +29,30 @@ modalBtnNextPage.forEach((modalBtnNextPage) => {
       currentStep += 1;
     }
   });
-});
\ No newline at end of file
+});
+
+/* FETCH API BOOKING 1 */
+const dateField = document.querySelector(".modal__date");
+dateField.setAttribute("min", new Date().toISOString().split('T')[0]);
+let timeSlots;
+async function getAvailableTimes(date) {
```

Comment:
Nice function! I would add another parameter, for the challenge ID. Even if you're hardcoding the challenge ID at this point in time, it's good practice to do the hard-coding as far "up" in the code as possible". By "up", I mean a place that is near where the code starts executing.

In this case, `getAvailableTimes()` is quite far "down". It gets executed by something else. The code that executes it is on line 57, which is further "up". It's not further up in the file, but it's further up in the chain of responsibilities.

So I would add a parameter here called `challengeId`, use that when calling the function on line 57, and hard-code the value there instead, see next comment.